nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/access-policies/ui/common/add-tenant-to-policy-dialog/add-tenant-to-policy-dialog.component.html

```diff
@@ -42,7 +42,7 @@ <h2 mat-dialog-title>Add Users/Groups To Policy</h2>
         </div>
     </mat-dialog-content>
     <mat-dialog-actions align="end" *ngIf="{ value: (saving$ | async)! } as saving">
-        <button mat-raised-button mat-dialog-close color="accent">Cancel</button>
+        <button mat-raised-button mat-dialog-close color="primary">Cancel</button>
```

Comment:
we shouldn't have 2 primary buttons on the same dialog. this one should probably be:
```suggestion
        <button mat-stroked-button mat-dialog-close color="primary">Cancel</button>
```